temperature.js

```diff
@@ -0,0 +1,16 @@
+let Temperature = {
+    toFahrenheit: function (celsius) {
```

Comment:
For using the rest of `es6` i'm confused as to why you used the `function` keyword here.